docs/deployment.md

```diff
@@ -4,9 +4,11 @@ The apply build and release process is split into two separate GitHub Actions pi
 
 - [Build](https://github.com/DFE-Digital/apply-for-teacher-training/actions/workflows/build.yml): This is the main CI pipeline which will automatically trigger a build from a commit to the `main` branch, this pipeline is also run on the merged PR head commit when a pull request is raised targetting `main`. If run from `main` branch this also triggers the deployment pipeline after all tests have succeeded.
 
-- [Deploy](https://github.com/DFE-Digital/apply-for-teacher-training/actions/workflows/deploy.yml): This is the main release pipeline that is used to deploy to all environments. Releases to `production` are triggered manually and the target environments can be chosen prior to deployment.
+- [Deploy](https://github.com/DFE-Digital/apply-for-teacher-training/actions/workflows/deploy.yml): This is the main release pipeline that is used to deploy to all environments. 
+    - Pull Requests that have the `deploy` label applied will trigger the deployment of a short lived Review App to its own self contained environment.  This provides an opportunity to manually review the changes prior to merging them to `main`, once the PR has been merged the Review App and its environment will be deleted.  
```

Comment:
What's "short-lived"? Does it delete itself?

Reply:
Yes, there's a workflow that runs when the PR is merged that deletes the Review App.  The last sentence is trying to explain that, does it need rewording?